core/src/main/java/org/arquillian/cube/impl/client/ForceStopDockerContainersShutdownHook.java

```diff
@@ -1,21 +1,35 @@
 package org.arquillian.cube.impl.client;
 
-import java.util.List;
 import org.arquillian.cube.spi.Cube;
 import org.arquillian.cube.spi.CubeRegistry;
 import org.jboss.arquillian.core.api.annotation.Observes;
 import org.jboss.arquillian.test.spi.event.suite.BeforeSuite;
 
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 public class ForceStopDockerContainersShutdownHook {
 
     public void attachShutDownHookForceStopDockerContainers(@Observes(precedence = 200) BeforeSuite event,
-        final CubeRegistry cubeRegistry) {
+                                                            final CubeRegistry cubeRegistry) {
+
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
             final List<Cube<?>> cubes = cubeRegistry.getCubes();
-            for (Cube cube : cubes) {
+            for (Cube cube: cubes) {
 
+                try {
                     cube.stop();
-                    cube.destroy();
+                } catch (final Throwable e) {
+                    Logger.getLogger(ForceStopDockerContainersShutdownHook.class.getName()).log(Level.WARNING, "Falied to stop: " + cube.getId(), e);
```

Comment:
how about declaring logger as `private static Logger logger = Logger.getLogger(ForceStopDockerContainersShutdownHook.class.getName());` and using it here.

Also there is typo for `Falied` -> `Failed`

Reply:
I'd only declare a logger static if I intend to use it more than once. Here it's only ever going to be used on an error that we don't expect to get often.